### DISCUSSION TITLE
Welcome to spglib Discussions!

### DISCUSSION THREAD

**Discussion by atztogo** [spglib/spglib#116] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qz844** [comment]
Hello,

I'm a college student currently working on some structures prediction project. I'm sorry if my questions are too naive.
Do I understand correctly that spglib works as a post analysis tool only for those crystal structures already optimized with ab-initio calculations?
Can spglib generate a structure with a particular space group from a single atom or molecular?

Thanks in advance!

**Reply by jochym** [reply]
There are other tools for that. You can use ASE library function `ase.spacegroup.crystal` for this task.
